cmd/apiserver/app/server/server.go

```diff
@@ -73,6 +77,9 @@ func NewCommandServer(out io.Writer) *cobra.Command {
 	// Store resources in etcd under our special prefix
 	options.EtcdOptions.StorageConfig.Prefix = etcdPathPrefix
 
+	// Set generated SSL cert path correctly
+	options.SecureServingOptions.ServerCert.CertDirectory = certDirectory
```

Comment:
I think there's a way to 'override' an existing flag, but I don't know the syntax offhand.